examples/helloworld.er

```diff
@@ -1,3 +1,6 @@
+sys = pyimport "sys"
+sys.stdout.reconfigure!(encoding:="utf-8")
```

Comment:
It is inconvenient that multilingual support is only available if the user sets it up using `sys.stdout.reconfigure!`, so I think it would be better to include a process to embed this code when generating the code.
For example, Erg embeds a code in the bytecode that passes its own standard API path and loads it. Using this as a reference, it is possible to call `sys.stdout.reconfigure` first.

https://github.com/erg-lang/erg/blob/b2fb80c41c877be449a6034adc0c50d3d7f9918f/crates/erg_compiler/codegen.rs#L3929-L3953